src/css/core.scss

```
@@ -322,8 +322,10 @@ nav {
 		height: 27px;
 	}
 
-	&:hover > ul {
+	&:hover > ul,
```

Currently, sublinks under the 'Admin' heading are reachable with the Tab key, but as soon as the Admin heading loses focus they become invisible again. There's no way of dealing with this in Edge or IE11 (of course), but you can make use of the `focus-within` pseudo-selector to keep everything visible in any modern web browser.

```suggestion
	&:focus-within > ul,
	&:hover > ul,
```